app/src/main/java/com/github/sdpteam15/polyevents/ItemsAdminActivity.kt

```diff
@@ -39,9 +36,14 @@ class ItemsAdminActivity : AppCompatActivity() {
             if (!it.value)
                 println("query not satisfied")
         }
+        currentDatabase.itemDatabase!!.getItemTypes(itemTypes).observe(this) {
+            if (!it.value)
+                println("query not satisfied")
```

Comment:
Try to avoid these debug prints and instead make a useful toast that will be shown to the user (I fix one of them in my branch and I can fix the other once you merged with master, it's for the future :) )

Reply:
Good point, I just changed it